hw2/05-form.js

```diff
@@ -1 +1,27 @@
 // Add your code here
+const form = document.getElementById("form");
+form.addEventListener("submit", function (e) {
+  //prevent the normal submission of the form
+  e.preventDefault();
+
+  // Get values from form fields
+  const fullName = document.querySelector("#fullname").value;
+  const email = document.querySelector("#email").value;
+  const registrationStatus = document.querySelector("#registration").value;
+
+  // Get values of the checkboxes
+  const programmingCheckbox = document.querySelector("#programming").checked;
+  const osCheckbox = document.querySelector("#os").checked;
+  const fullstackCheckbox = document.querySelector("#fullstack").checked;
+
+  const comment = document.querySelector("#comment").value;
+
+  // Print the collected values to the console
+  console.log("Full Name:", fullName);
+  console.log("Email:", email);
+  console.log("Registration Status:", registrationStatus);
+  console.log("Programming Checkbox:", programmingCheckbox);
```

Comment:
For these checkbox values, I think we could display results without explicitly mentioning that they are checkboxes, thinking about if we sent these results to someone and they don't know what that means. Maybe adding a title above the three courses that says "Classes Taken" and then displaying the course title next to true/false value or something like that.

Reply:
Thank you, I don't know how I missed this! I changed it.